### PR TITLE
Stop using NormalizeResources in bootstrap policy

### LINKED ISSUE #15821: RBAC Migration Followup #9: Remove normalizeResources from bootstrappolicy
Remove `normalizeResources` from `pkg/cmd/server/bootstrappolicy/policy.go`

### PR DESCRIPTION
**Pull request by simo5**
Just removed it.

Fixes #15821

### CODE REVIEW

File: pkg/cmd/server/bootstrappolicy/policy.go [hunk 1]
```diff
@@ -112,14 +112,6 @@ func GetBootstrapOpenshiftRoles(openshiftNamespace string) []rbac.Role {
...
 
-	// we don't want to expose the resourcegroups externally because it makes it very difficult for customers to learn from
-	// our default roles and hard for them to reason about what power they are granting their users
-	for i := range roles {
-		for j := range roles[i].Rules {
-			roles[i].Rules[j].Resources = authorizationapi.NormalizeResources(sets.NewString(roles[i].Rules[j].Resources...)).List()
-		}
-	}
-
 	return roles
```
Comment (enj) [inline pkg/cmd/server/bootstrappolicy/policy.go]: Remove the `roles` var and just `return []rbac.Role{` at the top.

Comment (simo5) [inline pkg/cmd/server/bootstrappolicy/policy.go]: done

**Comment by enj** [comment]
/lgtm
/test verify

Test flaked on SSH issue.

**Comment by enj** [comment]
/retest

Flaked on #15912

**Pull request merged**